### PR TITLE
records: message on successfully access

### PR DESCRIPTION
**Pull request by hachreak**
* Shows a message to know why the user has access to the record.
  (closes zenodo/zenodo-accessrequests#6)

### CODE REVIEW

**Comment by hachreak** [comment]
ping @lnielsen

**Comment by hachreak** [comment]
ping @lnielsen

**Comment by lnielsen** [comment]
Tests are failing - can you have a look.

**Comment by hachreak** [comment]
@lnielsen now all tests are passing

File: zenodo/modules/records/permissions.py [hunk 1]
```diff
@@ -206,9 +206,11 @@ def deny(user, record):
 def has_read_permission(user, record):
     """Check if user has read access to the record."""
     # Allow if record is open access
+    reason = 'You have access because {0}'
+
     if AccessRight.get(record['access_right'], record.get('embargo_date')) \
             == AccessRight.OPEN:
-        return True
+        return _(reason.format('the record has open access'))
```
Comment (krzysztof) [inline zenodo/modules/records/permissions.py]: @lnielsen What is the preferred way to report the reason for access?

**Pull request closed without merging**